parsec/backend/memory/organization.py

```diff
@@ -103,7 +104,13 @@ async def stats(self, id: OrganizationID) -> OrganizationStats:
             if vlob_organization_id == id:
                 data_size += blockmeta.size
 
-        users = len(self._user_component._organizations[id].users)
+        users = 0
```

Comment:
out of curiosity I was wondering how to keep this without a for loop (you know... because.... reasons... 😄 )

```python
from functools import reduce
stats = reduce(lambda c, user: {'users': c['users'] + 1, 'outsiders': c['outsiders'] + int(user.profile == UserProfile.OUTSIDER)}, data, {'users': 0, 'outsiders': 0})
```
:/

So in the end I think the for loop is much more readable here !